src/App.css

```diff
@@ -1,38 +1,164 @@
-.App {
+:root {
+  --background: #202024;
+  --main-purple: rgba(62, 24, 165, 0.9);
+  --secondary-gray: rgb(100, 99, 99);
+  --dark-transparent-gray: rgba(70, 70, 70, 0.4);
+}
+
+body {
+  background-color: var(--background);
+}
+
+.logoutLink {
+  float: right;
+  color: white;
+  cursor: pointer;
+  justify-content: top;
+}
+
+.centeredDiv {
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 200px;
+  width: 320px;
+  background: var(--main-purple);
   text-align: center;
+  padding: 30px;
+  border-radius: 10%;
+}
+
+.link {
+  cursor: pointer;
+  font-weight: bold;
+  color: white;
+}
+
+.regularInput {
+  width: 14em;
+  border: 1px solid gray;
+  border-radius: 4px;
+  color: var(--secondary-gray);
+  margin: auto;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.whiteLabel {
+  color: white;
+  margin-right: 10px;
+}
+
+.smallBtn {
+  width: 70%;
+  margin: 5px;
+  padding: 5px;
+  font-weight: bold;
+  font-size: 110%;
+  background-color: white;
+  border: 1px solid var(--main-purple);
```

Comment:
@marijana-kolosek Zasto ne koristis jednostavno ovaj format #333333 za boju na primer?

Reply:
ako mislite na ovo sto pise white ili ono sa rgb(0,0,0) to koristim zato sto mi je tako lakse, ali ako je potrebno mogu i samo ovaj format kao #333333 da koristim

Reply:
@marijana-kolosek Ja napravim obicno neki fajl colors ili constants odakle importujem boje onda.